packages/agent/src/invitation/handler.ts

```diff
@@ -32,15 +21,26 @@ import {
   parseMessageType,
 } from '@credo-ts/didcomm'
 import { supportsIncomingMessageType } from '@credo-ts/didcomm/build/util/messageType'
+import type {
+  OpenId4VciCredentialConfigurationSupportedWithFormats,
+  OpenId4VciDpopRequestOptions,
+  OpenId4VciRequestTokenResponse,
+  OpenId4VciResolvedAuthorizationRequest,
+  OpenId4VciResolvedCredentialOffer,
+} from '@credo-ts/openid4vc'
 import {
   getOfferedCredentials,
   getScopesFromCredentialConfigurationsSupported,
   preAuthorizedCodeGrantIdentifier,
 } from '@credo-ts/openid4vc'
-import { type Observable, filter, first, firstValueFrom, timeout } from 'rxjs'
-
+import { getOid4vcCallbacks } from '@credo-ts/openid4vc/build/shared/callbacks'
+import { getTrustedEntities } from '@easypid/utils/trustMechanism'
```

Comment:
We shouldn't depend on easypid from the agent package  (and will lead to issues once we start the SDK work). 

I think need to provide the config from the constants to the calls to agent, that way we also keep the flexibility to dynamically change the trusted entities

(so i think we should move the `getTrustedEntities` to the agent, but keep the config/constants in the easypid and provide that to the `getTrustedEntities` as function param)